controller/controller.py

```diff
@@ -8,8 +9,11 @@ class Controller(object):
     def __init__(self, driver):
         self.driver = driver
 
-    def open_chrome(self):
-        self.driver = webdriver.Chrome(executable_path='/Users/ilionailiadhi/Ora/automation-project/chromedriver')
+    def open_browser(self, browser):
+        if browser == 'firefox':
+            self.driver = webdriver.Firefox(executable_path='/Users/ilionailiadhi/Ora/automation-project/geckodriver')
```

Comment:
You are still using local paths, check the repo that I shared to change it to a path relative to your project